converters/fromarray.ts

```diff
@@ -0,0 +1,59 @@
+import {  IEvent, IEventArray, ILayoutState } from "../clarity";
+import { Action, ObjectType } from "../clarity";
+import schemas from "./schema";
+
+export default function(eventArray: IEventArray): IEvent {
+  let id          = eventArray[0];
+  let type        = eventArray[1];
+  let time        = eventArray[2];
+  let stateArray  = eventArray[3];
+  let schema      = eventArray[4];
```

Comment:
Can you used a destructuring assignment like

['id, type, time, stateArray, scheam] = eventArray?

?